app/templates/client/app/scripts/controllers/login.js

```diff
@@ -8,7 +8,14 @@
  * Controller of the clientApp
  */
 angular.module('clientApp')
-  .controller('LoginCtrl', function ($scope, Auth, $state) {
+  .controller('LoginCtrl', function ($scope, Auth, $state, Account, accessToken) {
```

Comment:
@ordavidil Please remove this injection of ```accessToken``` (Not needed) and replace it with the ```emailVerified```.